README.md

```diff
@@ -9,7 +9,17 @@ are related to this charm.
 This charm is a component of Charmed Kubernetes. For full information,
 please visit the [official Charmed Kubernetes docs](https://www.ubuntu.com/kubernetes/docs/charm-openstack-integrator).
 
+## Nagios
 
+This charm uses NRPE checks that are able to monitor servers, networks, subnets, floating-ips, security-groups and
+ports. These checks can be set through configuration using the "nrpe-<interface_type>-ids" (e.g. "nrpe-server-ids")
+option to monitor specific interfaces defined by IDs. The "nrpe-<interface_type>-ids" option could be defined as "all",
+however it is only allowed for servers networks, floating-ips and ports. Along with option "nrpe-<interface_type>-ids"
+set to value "all" it's possible to used "nrpe-skip-<interface_type>-ids" to set up interfaces, which will be skipped.
+
+The script itself to check the OpenStack interface also supports the possibility to use the select parameter,
+which serves as a filter. (`--select name=data-port` can be used to filter out ports named "data-port")
+However this option is not configurable through the configuration.
```

Comment:
Suggestion: The nrpe script implements the select parameter for ports (`--select key=value`), which can help filter by the value of a port property (e.g. `--select name=data-port`). However, this option is not configurable via Juju, hence it remains unsupported in the charm.

Reply:
> Suggestion: The nrpe script implements the select parameter for ports (`--select key=value`), which can help filter by the value of a port property (e.g. `--select name=data-port`). However, this option is not configurable via Juju, hence it remains unsupported in the charm.

I'll just change "port property" to "interface property", because it can be used with any of the above interfaces.